src/components/form/HdInputPassword.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div
+    :class="fieldClasses"
+    class="field field--input"
+  >
+    <img
+      v-if="icon"
+      :src="icon"
+      role="presentation"
+      class="field__icon"
+    >
+    <input
```

Comment:
Can we reuse our `HdInput` here?

Reply:
I didn't reuse HdInput because from what I understood from the issue is that we want to keep the password input logic out of it.

Reply:
We can reuse `HdInput` and avoid all the duplicated code from methods/computed. The idea is to remove the logic from the `HdInput` with this visibility toggle thing and so on.

Reply:
:+1: 

Can you update accordingly @aym3nb ? (when you can :))